src/context/ChatProvider.tsx

```diff
@@ -1,11 +1,36 @@
-import React, { createContext, useState, useContext } from 'react'
+import { useVeramo } from '@veramo-community/veramo-react'
+import { IDIDManager } from '@veramo/core'
+import React, { createContext, useState, useContext, useEffect } from 'react'
+import { pickup } from '../utils/didcomm-mediation'
 
 const ChatContext = createContext<any>({})
 
 const ChatProvider = (props: any) => {
   const [selectedDid, setSelectedDid] = useState()
   const [composing, setComposing] = useState(false)
   const [newRecipient, setNewRecipient] = useState()
+
+  const MINUTE_MS = 6000
+  const { agent } = useVeramo<IDIDManager>()
+  const checkMyDIDs = async () => {
```

Comment:
this section is flaky. Depending on how long the timeout was, or even if the console logs are included, I was getting different behavior (sometimes myDIDs is empty even when it shouldn't be). There's probably a better way to do this..